tests/integration/main.rs

```diff
@@ -0,0 +1,93 @@
+mod artichoke;
+
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
+use serde::ser::{Serialize, Serializer, SerializeStruct};
+use bstr::BString;
```

Comment:
I think for the changes below we'll also need to pull in `ByteSlice`.

```suggestion
use bstr::{BString, ByteSlice};
```